docs/api/I13nUtils.md

```diff
@@ -0,0 +1,40 @@
+## I13nUtils
+
+The `I13Utils` mixin provides util functions for you to access the i13n nodes and execute i13n functionalities
```

Comment:
Sure, thats ok. So whats the end result here? Will I13Utils just become the mixin now?